src/js/login/containers/Main.jsx

```diff
@@ -49,15 +52,15 @@ class Main extends React.Component {
   }
 
   getLogoutUrl() {
-    $.ajax({
-      url: `${environment.API_URL}/v0/sessions`,
-      type: 'DELETE',
-      headers: {
+    fetch(`${environment.API_URL}/v0/sessions`, {
+      method: 'DELETE',
+      headers: new Headers({
         Authorization: `Token token=${sessionStorage.userToken}`
-      },
-      success: (result) => {
-        this.setState({ logoutUrl: result.logout_via_get });
-      }
+      })
+    }).then(response => {
```

Comment:
Should we handle errors in these calls? I think if there's an http error now we're going to generate some JS errors, instead of just failing silently.